docs/index.rst

```diff
@@ -1,18 +1,118 @@
+
+3. Bridge Networking vs. Host Networking
+
+	Bridge networking is currently only supported on a single host.  For multiple hosts, you will need to use overlay networks which are not currently supported. It order to expose Kafka to clients outside of the bridge network, you need to find the container IP and put it in ``advertised.listeners``.  This can be difficult to achieve depending on how you're using the images.  Furthermore, it can add a network hop and may not be as performant as the host network, which shares the network stack.  In summary, host networking is the recommended option in the following cases:
+
+		* Multi-host clusters without using Swarm/Kubernetes host network is the best approach
+		* If you need clients to be able to access Kafka outside the bridge/overlay network
+
+4. Launch Settings
+
+    Docker containers should be launched with ``Restart=always`` unless you are using a process manager.   This ensures that intermittent failures in the Docker environment do not result in unnecessary failures of the Confluent services.
+
+5. Adding Connectors to the Kafka Connect Image
+
+	There are currently two ways to add new connectors to the Kafka Connect image.
+
+	* Build a new Docker image that has the connector installed. You can follow the examples found in `Extending Images <development.html#extending-the-docker-images>`_. You will need to make sure that the connector jars are on the CLASSPATH for the Connect service (the default location of /usr/share/java/kafka-connect-* is the recommended location).
+	* Add the connector jars via volumes.  If you don't want to create a new Docker image, please see our documentation on `Configuring Kafka Connect with External Jars <operations/external-volumes.html>`_ to configure the `cp-kafka-connect` container with external jars.
+
+6. Included Java
+
+    The Confluent Docker images are tested and shipped with `Azul Zulu OpenJDK <https://www.azul.com/products/zulu/>`_.  Other JDK's (including Oracle Java) are supported, but you must extend the images yourself to implement that change.
+
+7. Untested Features
+
+	The following features/environments are not currently tested:
+
+		* Schema Registry SSL
+		* Kafka Connect with Security Enabled
+		* Confluent Control Center with Security Enabled
+		* The images are not currently tested on Docker Swarm.
+
+License
+-------
+
+The Confluent Platform Docker Images are available as open source software under the Apache License v2.0 license.  License details for the individual Confluent Platform components packaged in the images are available with the top-level `platform documentation <http://docs.confluent.io/current/platform.html>`_.
 
 Contents:
 
 .. toctree::
-   :maxdepth: 3
```

Comment:
seems odd that we'd be moving the table of contents to the bottom of the page

Reply:
@xvrl The table of contents is already shown on the left-hand navigation panel - this in-page contents is secondary so i don't think it should be the primary focus of the page. Actually it can probably be removed entirely. What do you think?   

![image](https://user-images.githubusercontent.com/11722533/32740263-39e9432c-c857-11e7-843b-8684029dfbd4.png)

Reply:
ping @xvrl ^

Reply:
we can probably remove it, though it looks like it used to be right below the title, and this pushes it to the bottom of the page. We should probably do whichever is more consistent with our other pages.